app/src/main/java/uy/com/bix/app/smsproject/activity/MainActivity.java

```diff
@@ -14,6 +22,29 @@
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
+
+		Log.d("Main Activity", "This is my message");
```

Comment:
Maybe you are using this for debugging, but if it isn't necessary we could delete it :)